src/main/java/beam/analysis/plots/GraphUtils.java

```diff
@@ -12,38 +12,46 @@
 import org.jfree.data.xy.XYDataset;
 import org.jfree.data.xy.XYSeries;
 import org.jfree.data.xy.XYSeriesCollection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 import java.awt.*;
 import java.io.File;
 import java.io.IOException;
 import java.util.*;
 import java.util.List;
+import java.util.Map;
+import java.util.Random;
 
 public class GraphUtils {
-    private static final Map<String, Color> colorsForModes = new HashMap<>();
-    private static final List<Color> colors = new ArrayList<>();
+    private static final Logger log = LoggerFactory.getLogger(GraphUtils.class);
+
     private static final Color DEFAULT_BACK_GROUND = new Color(255, 255, 255);
-    private static final Color PURPLE = new Color(102,0,153);
-    private static final Color LIGHT_BROWN = new Color(153,102,0);
-    private static final Color LIGHT_YELLOW = new Color(255, 255,153);
+    private static final Color PURPLE = new Color(102, 0, 153);
+    private static final Color LIGHT_BROWN = new Color(153, 102, 0);
+    private static final Color LIGHT_YELLOW = new Color(255, 255, 153);
     private static final Color VERY_LIGHT_BLUE = new Color(51, 204, 255);
-    private static final Color VERY_LIGHT_RED = new Color(255, 102,102);
-    private static final Color VERY_LIGHT_GREEN = new Color(102,255,102);
-    private static final Color VERY_DARK_BLUE = new Color(0,0,153);
-    private static final Color VERY_DARK_RED = new Color(153,0,0);
-    private static final Color VERY_DARK_GREEN = new Color(0,102,0);
-    private static final Color OLIVE = new Color(107,142,35);
-    private static final Color THISTLE = new Color(216,191,216);
-    private static final Color CADETBLUE = new Color(95,158,160);
+    private static final Color VERY_LIGHT_RED = new Color(255, 102, 102);
+    private static final Color VERY_LIGHT_GREEN = new Color(102, 255, 102);
+    private static final Color VERY_DARK_BLUE = new Color(0, 0, 153);
+    private static final Color VERY_DARK_RED = new Color(153, 0, 0);
+    private static final Color VERY_DARK_GREEN = new Color(0, 102, 0);
+    private static final Color OLIVE = new Color(107, 142, 35);
+    private static final Color THISTLE = new Color(216, 191, 216);
+    private static final Color CADETBLUE = new Color(95, 158, 160);
     private static final Color DARK_PINK = new Color(255, 0, 227);
 
+    private static final Map<String, Color> colorsForModes = new HashMap<>();
+    public static final Map<String, Color> carTypesColors = new HashMap<>();
```

Comment:
@dgm90 if `carTypesColors` is not accessed outside this class then change its visibility from `public` to `private`

Reply:
We access from `CarTripStatsFromPathTraversalEventHandler`, so yes, we need a public access here.